FrEIA/modules/splines/binned.py

```diff
@@ -75,6 +75,7 @@ def __init__(self, dims_in, dims_c=None, bins: int = 10, parameter_counts: Dict[
             default_domain: tuple of (left, right, bottom, top) default spline domain values
                 these values will be used as the starting domain (when the network outputs zero)
             identity_tails: whether to use identity tails for the spline
+            domain_clamping: clamping value for the domain
```

Comment:
Docstring could be more explicit -- how about "if float, clamp spline domain to (-domain_clamping, domain_clamping)"